sunpy/database/tests/test_database.py

```diff
@@ -441,6 +445,52 @@ def test_add_entry_from_hek_qr(database):
     assert len(database) > 1
 
 
+@pytest.mark.remote_data
```

Comment:
Do we still need this?

Reply:
Yes this is for search

Reply:
And also for h2vclient used to convert hek query to vso query